scripts/docker/Dockerfile-build

```diff
@@ -1,24 +1,32 @@
-FROM golang:1.8.4
+ARG base_image=opensuse:latest
 
-ENV GOPATH /go
-ENV GOBIN /go/bin
+FROM ${base_image}
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+RUN zypper --non-interactive addrepo --check --gpgcheck obs://Cloud:Tools Cloud:Tools
+RUN zypper --non-interactive addrepo --check --gpgcheck obs://devel:languages:go devel:languages:go
+RUN zypper --non-interactive --gpg-auto-import-keys refresh
+RUN zypper --non-interactive install \
+    cf-cli \
     git \
-    bash \
+    go \
     make \
-    gcc \
-    mercurial \
-    build-essential \
-    && apt-get autoremove -y \
-    && apt-get clean -y \
```

Comment:
The `zypper` equivalent would be something like

```
zypper --non-interactive clean --all
```

Is the image so big anyways that this kind of cleanup is pointless?

Reply:
Nope; it's just that we never actually ship this image anywhere, so cleanup is pointless.  We push images that are _not_ based on this image (but copy files out of results).